src/main/resources/db/migration/V1.1.9__Gir_teamet_flere_rettigheter.sql

```diff
@@ -0,0 +1,12 @@
+DO
+$$
+    BEGIN
+        IF EXISTS
+            (SELECT 1 from pg_roles where rolname = 'cloudsqliamuser')
+        THEN
+            GRANT USAGE ON SCHEMA public TO cloudsqliamuser;
+            GRANT ALL ON ALL TABLES IN SCHEMA public TO cloudsqliamuser;
+            ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL ON TABLES TO cloudsqliamuser;
```

Comment:
Måtte sette disse til GRANT ALL igjen fordi jeg måtte slette all data i basen 🙈 Kom også på at jeg måtte oppdatere systembruker mappingen

Reply:
Vil det være nok med GRANT DELETE eventuelt?

Reply:
Jeg støtter det at vi har begrenset med skrivemuligheter. Kunne f.eks. standard vært kun les, og helle heve det midlertidig mens du hadde behov for å slette noe?

Reply:
Ja, jeg er enig at vi burde ha begrenset skrivemuligheter 😊  Ble bare litt usikker på hvordan vi skulle gjøre det med systembruker mappingen tabellen? Da må vi legge til et nytt flyway script hver gang vi skal legge til en ny produsent? Vi trenger ikke ha GRANT ALL. Kan ha INSERT og DELETE nå, men legge til et nytt flyway script uten DELETE når vi er ferdig med test? 
Jeg tenkte det var greit med GRANT ALL nå frem til test var ferdig. Så vi slipper å ha så mange flyway filer, men jeg er absolutt åpen for andre måter å gjøre dette på 👍

Reply:
Skjønner. Jeg synes egentlig at vi skulle vurdert å legge til produsent-mappinger via Flyway, da slipper vi manuell tukling i databasen.

Reply:
Ikke noe problem å ha litt utvidede muligheter nå under utvikling, men vi må bare huske at vi faktisk fjerner det senere. Kanskje lage en jira-oppgave på det alt nå?

Reply:
Jepp, jeg lager to jira-oppgaver på rettigheter og endre produsent-mapping via flyway 😊 👍